goethe/src/test/java/com/palantir/goethe/GoetheBootstrapTest.java

```diff
@@ -87,8 +87,29 @@ public void testFormatting(FormatterFacade formatter) {
                         + "class Foo {\n"
                         + "    static {\n"
                         + "        System.out.println(\n"
-                        + "                \"" + padding + "\");\n"
+                        + "                \"" + longWord + "\");\n"
                         + "    }\n"
                         + "}\n");
     }
+
+    @ParameterizedTest
+    @MethodSource("formatterFacades")
+    public void testFormatJavadoc(FormatterFacade formatter) {
+        String longWord = "a".repeat(90);
+        JavaFile javaFile = JavaFile.builder(
+                        "com.palantir.foo",
+                        TypeSpec.classBuilder("Foo")
+                                .addJavadoc("$1L $1L", longWord)
+                                .build())
+                .build();
+
+        assertThat(format(formatter, javaFile))
```

Comment:
Didn't include the following code from the above test because it's kind of pointless here: we know Goethe will reformat the non-Javadoc portions.
```
.as("Expected the formatted output to differ from original")
.isNotEqualTo(raw)
```